sources/europe/no/Norway-ortho2.geojson

```diff
@@ -0,0 +1,55 @@
+{
+    "type": "Feature",
+    "properties": {
+        "best": true, 
+        "type": "tms",
+        "name": "Norway Orthophoto (more recent, less zoom)",
+        "id": "geovekst-nib2",
+        "country_code": "NO",
+        "description": "Most recent Norwegian orthophotos (max zoom 19), courtesy of Geovekst and Kartverket.",
+        "url": "https://opencache{switch:,2,3}.statkart.no/gatekeeper/gk/gk.open_nib_web_mercator_wmts_v2?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=Nibcache_web_mercator_v2&STYLE=default&FORMAT=image/jpgpng&tileMatrixSet=default028mm&tileMatrix={zoom}&tileRow={y}&tileCol={x}",
+        "icon": "https://register.geonorge.no/data/organizations/_L_norgeibilder96x96.png",
+        "max_zoom": 21,
```

Comment:
the max-zoom is wrong no? @NKAmapper @grischard

Reply:
Hm, you are right, should be 19 of course. Thank you!